docs/pages/careers.tsx

```diff
@@ -147,16 +147,57 @@ const openRolesData = [
     title: 'Engineering',
     roles: [
       {
-        title: 'React Engineer - X',
+        title: 'React Tech Lead - Core',
         description:
-          'You will strengthen the advanced components team, build new ambitious complex features, work on strategic problems, and help grow the adoption.',
-        url: '/careers/react-engineer-x/',
+          'You will lead the development of MUI Core, positioning the library as the industry standard for design teams while doubling the adoption of the library.',
+        url: '/careers/react-tech-lead-core/',
+      },
+      {
+        title: 'React Engineer - Core',
+        description:
+          'You will strengthen the core components team, e.g. collaborate with the community to land contributions.',
```

Comment:
To remove "e.g." - best to avoid these kinds of Latinisms as a general rule.

```suggestion
          'You will strengthen the core components team by collaborating with the community to land contributions.',
```